### PR TITLE
l-loader fixes2

### PR DESCRIPTION
**Pull request by vchong**
@fboudra fyi

### CODE REVIEW

File: BL33_AP_UEFI.fd [hunk 1]
```diff
@@ -0,0 +1 @@
+/home/victor.chong/work/swg/repo_960/build/../edk2/Build/HiKey960/RELEASE_GCC49/FV/BL33_AP_UEFI.fd
```
Comment (fboudra) [inline BL33_AP_UEFI.fd]: seems wrong

Comment (vchong) [reply BL33_AP_UEFI.fd]: Oops sorry! This got added in by accident. Will push a new commit.

Comment (vchong) [reply BL33_AP_UEFI.fd]: File removed. Thanks for noticing!

**Comment by fboudra** [comment]
LGTM.

**Pull request merged**